FetchRecipe.js

```diff
@@ -0,0 +1,16 @@
+function addRandomRecipe() {
+  const recipes =
```

Comment:
Great start!

I think the next step (for a new PR) is to generalize this formatting into a function that takes a recipe object as an input:

```
{
“id”: 0 
“link”: “some link”,
“title”: “some title”,
“like”: 5,
“category”: “some category”
}
```

And returns an HTML block that's formatted how you would like to see a recipe in the webpage. You could imagine the category's page might use it like this (pseudocode):
```
getRecipesByCategory() {
     var recipes = fetch('category-servlet');
     var recipeList = []
     for recipe in recipes {
         var formatted = formatRecipeToHTML(recipe);
         recipeList.add(formatted);
}
document.getElementById('category-page-recipe-list').innerText = recipeLIst;
```

Similarly the frontpage might call the function too with different recipes. With a convenient formatting function, we can make sure that recipes look the same no matter where they are applied. Your function could also tag the HTML block with a CSS class so you can add styling through that too.